packages/components/src/components/CoinLogo/index.native.tsx

```diff
@@ -31,12 +31,20 @@ const Logo = styled.Image``;
 
 interface Props {
     network: string;
+    height: number | string;
 }
 
-const CoinLogo = ({ network, ...rest }: Props) => {
+const CoinLogo = ({ network, height = 50, ...rest }: Props) => {
     return (
-        // eslint-disable-next-line import/no-dynamic-require(
-        <Logo source={LOGOS.xtz} {...rest} />
+        <Logo
+            source={LOGOS[network]}
+            style={{
+                width: height,
+                height,
+            }}
+            resizeMode="contain"
```

Comment:
it should work out of the box with this line!
https://stackoverflow.com/a/51897371

Reply:
This solution doesn't work you need to set the height too. Otherwise there is nothing rendered 0px height.

Reply:
works on my machine^TM

Reply:
I'm working on SVG logos on different branch, so this resize mode doesn't matter anymore.